lib/impl/AzureRevokePresignTvm.js

```diff
@@ -37,12 +37,17 @@ class AzureRevokePresignTvm extends Tvm {
    */
   async _generateCredentials (params) {
     const containerName = Tvm._hash(params.owNamespace)
+    const containerNamePublic = Tvm._hash(params.owNamespace) + '-public'
+
     const accountURL = `https://${params.azureStorageAccount}.blob.core.windows.net`
 
     const sharedKeyCredential = new azure.SharedKeyCredential(params.azureStorageAccount, params.azureStorageAccessKey)
     const containerURL = azureUtil.getContainerURL(accountURL, sharedKeyCredential, containerName)
+    const containerURLPublic = azureUtil.getContainerURL(accountURL, sharedKeyCredential, containerNamePublic)
 
+    // revoke presign urls by resetting the default access policy
     await azureUtil.setAccessPolicy(containerURL, params.azureStorageAccount)
+    await azureUtil.setAccessPolicy(containerURLPublic, params.azureStorageAccount, true)
```

Comment:
the function only takes 2 params?

Reply:
yes that's the util see https://github.com/adobe/aio-tvm/pull/194/files#diff-40ac97e39d53d7b1cd71c91052d541fe2a0a3dd66aca8a781a869b0960749796R90

Reply:
uff could have caught that `params.azureStorageAccount` has nothing to do here, see #195